packages/@aws-cdk/aws-certificatemanager/lambda-packages/dns_validated_certificate_handler/test/handler.test.js

```diff
@@ -145,10 +148,103 @@ describe('DNS Validated Certificate Handler', () => {
       });
   });
 
+  test('Create operation with SubjectAlternativeNames property requests a certificate with multiple DNS records', () => {
+    const requestCertificateFake = sinon.fake.resolves({
+      CertificateArn: testCertificateArn
+    });
+
+    const describeCertificateFake = sinon.stub();
+    describeCertificateFake.onFirstCall().resolves({
+      Certificate: {
+        CertificateArn: testCertificateArn
+      }
+    });
+    describeCertificateFake.resolves({
+      Certificate: {
+        CertificateArn: testCertificateArn,
+        DomainValidationOptions: [{
```

Comment:
Indent this like the following:

```ts
        DomainValidationOptions: [
          {
            ValidationStatus: 'SUCCESS',
            ResourceRecord: {
              Name: testRRName,
              Type: 'CNAME',
              Value: testRRValue,
            },
          },
          {
            ValidationStatus: 'SUCCESS',
            ResourceRecord: {
              Name: testAltRRName,
              Type: 'CNAME',
              Value: testAltRRValue,
            },
          },
        ],
```

Reply:
@skinny85 I have added a rule config to the .eslintrc.json of the lambda-package as follows:
```json
"comma-dangle": ["error", {
    "arrays": "always-multiline",
    "objects": "always-multiline",
     "imports": "never",
    "exports": "never",
    "functions": "never"
}]
```
And ran `eslint --fix` on both `lib/index.js` and `test/handler.test.js`. Let me know if this is what you want. This created a lot of mess, because `test/handler.test.js` did not follow the convention that you have requested that much....

Let me know if this is what you've wanted. Otherwise, tell me what the eslint config should be.

Reply:
I'd rather the `.eslintrc.json` file not be modified, and to minimize incidental changes to the lambda code not related to the logic changes.